kofta/src/app/useMainWsHandler.ts

```diff
@@ -417,6 +416,18 @@ export const useMainWsHandler = () => {
           wsend({ op: "get_current_room_users", d: {} });
         }
       },
+
+      // ! This won't work for some reason.
+      you_joined_waiting_room: (d) => {
```

Comment:
what do you mean it doesn't work?

Reply:
It doesnt react when the backend sends the message